GoogleDAIDemo/app/src/main/java/com/jwplayer/googledai/JWPlayerNativeControls.java

```diff
@@ -0,0 +1,249 @@
+package com.jwplayer.googledai;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.RelativeLayout;
+import android.widget.SeekBar;
+import android.widget.TextView;
+
+import com.longtailvideo.jwplayer.JWPlayerView;
+import com.longtailvideo.jwplayer.core.PlayerState;
+import com.longtailvideo.jwplayer.events.CompleteEvent;
+import com.longtailvideo.jwplayer.events.PauseEvent;
+import com.longtailvideo.jwplayer.events.PlayEvent;
+import com.longtailvideo.jwplayer.events.TimeEvent;
+import com.longtailvideo.jwplayer.events.listeners.VideoPlayerEvents;
+
+import java.util.ArrayList;
+
+public class JWPlayerNativeControls extends RelativeLayout
```

Comment:
@jlod9412 Do we need the native controls class? Lets use our original controls instead